_includes/usage/cf-core.html

```diff
@@ -10,13 +10,13 @@
 <ul>
 <li><a href="#variables">Variables</a>
 <ul>
-<li><a href="#sizing-variables">Sizing variables</a></li>
 <li><a href="#color-variables">Color variables</a></li>
+<li><a href="#sizing-variables">Sizing variables</a></li>
 </ul>
 </li>
 <li><a href="#media-queries">Media queries</a>
 <ul>
-<li><a href="#respond-to-min-and-max-mixins">Respond to min and max mixins</a></li>
+<li><a href="#respond-to-min-and-max-width-mixins">Respond to min and max mixins</a></li>
```

Comment:
Update the link text here, too? (But I guess that needs to happen in the `usage.md` in capital-framework.)